### DISCUSSION TITLE
ERROR: Could not find a version that satisfies the requirement boto3>=1.26.30

### DISCUSSION THREAD

**Discussion by eddie727** [jertel/elastalert2#1125] [Q&A] [answered]
env: amazon linux
installed: python3.11 openssl1.1.1 
i tried `pip install elastalert2` but `ERROR: Could not find a version that satisfies the requirement boto3>=1.26.30` came out
i figured out that there is no 1.26.30 version in elasalert2...
how to figure out it

`(from elastalert2) (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.0.12, 0.0.13, 0.0.14, 0.0.16, 0.0.17, 0.0.18, 0.0.19, 0.0.20, 0.0.21, 0.0.22, 1.0.0, 1.0.1, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.3.0, 1.3.1, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.5, 1.4.6, 1.4.7, 1.4.8, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5, 1.5.6, 1.5.7, 1.5.8, 1.5.9, 1.5.10, 1.5.11, 1.5.12, 1.5.13, 1.5.14, 1.5.15, 1.5.16, 1.5.17, 1.5.18, 1.5.19, 1.5.20, 1.5.21, 1.5.22, 1.5.23, 1.5.24, 1.5.25, 1.5.26, 1.5.27, 1.5.28, 1.5.29, 1.5.30, 1.5.31, 1.5.32, 1.5.33, 1.5.34, 1.5.35, 1.5.36, 1.6.0, 1.6.1, 1.6.2, 1.6.3, 1.6.4, 1.6.5, 1.6.6, 1.6.7, 1.6.8, 1.6.9, 1.6.10, 1.6.11, 1.6.12, 1.6.13, 1.6.14, 1.6.15, 1.6.16, 1.6.17, 1.6.18, 1.6.19, 1.6.20, 1.6.21, 1.6.22, 1.6.23, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.7.5, 1.7.6, 1.7.7, 1.7.8, 1.7.9, 1.7.10, 1.7.11, 1.7.12, 1.7.13, 1.7.14, 1.7.15, 1.7.16, 1.7.17, 1.7.18, 1.7.19, 1.7.20, 1.7.21, 1.7.22, 1.7.23, 1.7.24, 1.7.25, 1.7.26, 1.7.27, 1.7.28, 1.7.29, 1.7.30, 1.7.31, 1.7.32, 1.7.33, 1.7.34, 1.7.35, 1.7.36, 1.7.37, 1.7.38, 1.7.39, 1.7.40, 1.7.41, 1.7.42, 1.7.43, 1.7.44, 1.7.45, 1.7.46, 1.7.47, 1.7.48, 1.7.49, 1.7.50, 1.7.51, 1.7.52, 1.7.53, 1.7.54, 1.7.55, 1.7.56, 1.7.57, 1.7.58, 1.7.59, 1.7.60, 1.7.61, 1.7.62, 1.7.63, 1.7.64, 1.7.65, 1.7.66, 1.7.67, 1.7.68, 1.7.69, 1.7.70, 1.7.71, 1.7.72, 1.7.73, 1.7.74, 1.7.75, 1.7.76, 1.7.77, 1.7.78, 1.7.79, 1.7.80, 1.7.81, 1.7.82, 1.7.83, 1.7.84, 1.8.0, 1.8.1, 1.8.2, 1.8.3, 1.8.4, 1.8.5, 1.8.6, 1.8.7, 1.8.8, 1.8.9, 1.9.0, 1.9.1, 1.9.2, 1.9.3, 1.9.4, 1.9.5, 1.9.6, 1.9.7, 1.9.8, 1.9.9, 1.9.10, 1.9.11, 1.9.12, 1.9.13, 1.9.14, 1.9.15, 1.9.16, 1.9.17, 1.9.18, 1.9.19, 1.9.20, 1.9.21, 1.9.22, 1.9.23, 1.9.24, 1.9.25, 1.9.26, 1.9.27, 1.9.28, 1.9.29, 1.9.30, 1.9.31, 1.9.32, 1.9.33, 1.9.34, 1.9.35, 1.9.36, 1.9.37, 1.9.38, 1.9.39, 1.9.40, 1.9.41, 1.9.42, 1.9.43, 1.9.44, 1.9.45, 1.9.46, 1.9.47, 1.9.48, 1.9.49, 1.9.50, 1.9.51, 1.9.52, 1.9.53, 1.9.54, 1.9.55, 1.9.56, 1.9.57, 1.9.58, 1.9.59, 1.9.60, 1.9.61, 1.9.62, 1.9.63, 1.9.64, 1.9.65, 1.9.66, 1.9.67, 1.9.68, 1.9.69, 1.9.70, 1.9.71, 1.9.72, 1.9.73, 1.9.74, 1.9.75, 1.9.76, 1.9.77, 1.9.78, 1.9.79, 1.9.80, 1.9.81, 1.9.82, 1.9.83, 1.9.84, 1.9.85, 1.9.86, 1.9.87, 1.9.88, 1.9.89, 1.9.90, 1.9.91, 1.9.92, 1.9.93, 1.9.94, 1.9.95, 1.9.96, 1.9.97, 1.9.98, 1.9.99, 1.9.100, 1.9.101, 1.9.102, 1.9.103, 1.9.104, 1.9.105, 1.9.106, 1.9.107, 1.9.108, 1.9.109, 1.9.110, 1.9.111, 1.9.112, 1.9.113, 1.9.114, 1.9.115, 1.9.116, 1.9.117, 1.9.118, 1.9.119, 1.9.120, 1.9.121, 1.9.122, 1.9.123, 1.9.124, 1.9.125, 1.9.126, 1.9.127, 1.9.128, 1.9.129, 1.9.130, 1.9.131, 1.9.132, 1.9.133, 1.9.134, 1.9.135, 1.9.136, 1.9.137, 1.9.138, 1.9.139, 1.9.140, 1.9.141, 1.9.142, 1.9.143, 1.9.144, 1.9.145, 1.9.146, 1.9.147, 1.9.148, 1.9.149, 1.9.150, 1.9.151, 1.9.152, 1.9.153, 1.9.154, 1.9.155, 1.9.156, 1.9.157, 1.9.158, 1.9.159, 1.9.160, 1.9.161, 1.9.162, 1.9.163, 1.9.164, 1.9.165, 1.9.166, 1.9.167, 1.9.168, 1.9.169, 1.9.170, 1.9.171, 1.9.172, 1.9.173, 1.9.174, 1.9.175, 1.9.176, 1.9.177, 1.9.178, 1.9.179, 1.9.180, 1.9.181, 1.9.182, 1.9.183, 1.9.184, 1.9.185, 1.9.186, 1.9.187, 1.9.188, 1.9.189, 1.9.190, 1.9.191, 1.9.192, 1.9.193, 1.9.194, 1.9.195, 1.9.196, 1.9.197, 1.9.198, 1.9.199, 1.9.200, 1.9.201, 1.9.202, 1.9.203, 1.9.204, 1.9.205, 1.9.206, 1.9.207, 1.9.208, 1.9.209, 1.9.210, 1.9.211, 1.9.212, 1.9.213, 1.9.214, 1.9.215, 1.9.216, 1.9.217, 1.9.218, 1.9.219, 1.9.220, 1.9.221, 1.9.222, 1.9.223, 1.9.224, 1.9.225, 1.9.226, 1.9.227, 1.9.228, 1.9.229, 1.9.230, 1.9.231, 1.9.232, 1.9.233, 1.9.234, 1.9.235, 1.9.236, 1.9.237, 1.9.238, 1.9.239, 1.9.240, 1.9.241, 1.9.242, 1.9.243, 1.9.244, 1.9.245, 1.9.246, 1.9.247, 1.9.248, 1.9.249, 1.9.250, 1.9.251, 1.9.252, 1.9.253, 1.10.0, 1.10.1, 1.10.2, 1.10.3, 1.10.4, 1.10.5, 1.10.6, 1.10.7, 1.10.8, 1.10.9, 1.10.10, 1.10.11, 1.10.12, 1.10.13, 1.10.14, 1.10.15, 1.10.16, 1.10.17, 1.10.18, 1.10.19, 1.10.20, 1.10.21, 1.10.22, 1.10.23, 1.10.24, 1.10.25, 1.10.26, 1.10.27, 1.10.28, 1.10.29, 1.10.30, 1.10.31, 1.10.32, 1.10.33, 1.10.34, 1.10.35, 1.10.36, 1.10.37, 1.10.38, 1.10.39, 1.10.40, 1.10.41, 1.10.42, 1.10.43, 1.10.44, 1.10.45, 1.10.46, 1.10.47, 1.10.48, 1.10.49, 1.10.50, 1.11.0, 1.11.1, 1.11.2, 1.11.3, 1.11.4, 1.11.5, 1.11.6, 1.11.7, 1.11.8, 1.11.9, 1.11.10, 1.11.11, 1.11.12, 1.11.13, 1.11.14, 1.11.15, 1.11.16, 1.11.17, 1.12.0, 1.12.1, 1.12.2, 1.12.3, 1.12.4, 1.12.5, 1.12.6, 1.12.7, 1.12.8, 1.12.9, 1.12.10, 1.12.11, 1.12.12, 1.12.13, 1.12.14, 1.12.15, 1.12.16, 1.12.17, 1.12.18, 1.12.19, 1.12.20, 1.12.21, 1.12.22, 1.12.23, 1.12.24, 1.12.25, 1.12.26, 1.12.27, 1.12.28, 1.12.29, 1.12.30, 1.12.31, 1.12.32, 1.12.33, 1.12.34, 1.12.35, 1.12.36, 1.12.37, 1.12.38, 1.12.39, 1.12.40, 1.12.41, 1.12.42, 1.12.43, 1.12.44, 1.12.45, 1.12.46, 1.12.47, 1.12.48, 1.12.49, 1.13.0, 1.13.1, 1.13.2, 1.13.3, 1.13.4, 1.13.5, 1.13.6, 1.13.7, 1.13.8, 1.13.9, 1.13.10, 1.13.11, 1.13.12, 1.13.13, 1.13.14, 1.13.15, 1.13.16, 1.13.17, 1.13.18, 1.13.19, 1.13.20, 1.13.21, 1.13.22, 1.13.23, 1.13.24, 1.13.25, 1.13.26, 1.14.0, 1.14.1, 1.14.2, 1.14.3, 1.14.4, 1.14.5, 1.14.6, 1.14.7, 1.14.8, 1.14.9, 1.14.10, 1.14.11, 1.14.12, 1.14.13, 1.14.14, 1.14.15, 1.14.16, 1.14.17, 1.14.18, 1.14.19, 1.14.20, 1.14.21, 1.14.22, 1.14.23, 1.14.24, 1.14.25, 1.14.26, 1.14.27, 1.14.28, 1.14.29, 1.14.30, 1.14.31, 1.14.32, 1.14.33, 1.14.34, 1.14.35, 1.14.36, 1.14.37, 1.14.38, 1.14.39, 1.14.40, 1.14.41, 1.14.42, 1.14.43, 1.14.44, 1.14.45, 1.14.46, 1.14.47, 1.14.48, 1.14.49, 1.14.50, 1.14.51, 1.14.52, 1.14.53, 1.14.54, 1.14.55, 1.14.56, 1.14.57, 1.14.58, 1.14.59, 1.14.60, 1.14.61, 1.14.62, 1.14.63, 1.15.0, 1.15.1, 1.15.2, 1.15.3, 1.15.4, 1.15.5, 1.15.6, 1.15.7, 1.15.8, 1.15.9, 1.15.10, 1.15.11, 1.15.12, 1.15.13, 1.15.14, 1.15.15, 1.15.16, 1.15.17, 1.15.18, 1.16.0, 1.16.1, 1.16.2, 1.16.3, 1.16.4, 1.16.5, 1.16.6, 1.16.7, 1.16.8, 1.16.9, 1.16.10, 1.16.11, 1.16.12, 1.16.13, 1.16.14, 1.16.15, 1.16.16, 1.16.17, 1.16.18, 1.16.19, 1.16.20, 1.16.21, 1.16.22, 1.16.23, 1.16.24, 1.16.25, 1.16.26, 1.16.27, 1.16.28, 1.16.29, 1.16.30, 1.16.31, 1.16.32, 1.16.33, 1.16.34, 1.16.35, 1.16.36, 1.16.37, 1.16.38, 1.16.39, 1.16.40, 1.16.41, 1.16.42, 1.16.43, 1.16.44, 1.16.45, 1.16.46, 1.16.47, 1.16.48, 1.16.49, 1.16.50, 1.16.51, 1.16.52, 1.16.53, 1.16.54, 1.16.55, 1.16.56, 1.16.57, 1.16.58, 1.16.59, 1.16.60, 1.16.61, 1.16.62, 1.16.63, 1.17.0, 1.17.1, 1.17.2, 1.17.3, 1.17.4, 1.17.5, 1.17.6, 1.17.7, 1.17.8, 1.17.9, 1.17.10, 1.17.11, 1.17.12, 1.17.13, 1.17.14, 1.17.15, 1.17.16, 1.17.17, 1.17.18, 1.17.19, 1.17.20, 1.17.21, 1.17.22, 1.17.23, 1.17.24, 1.17.25, 1.17.26, 1.17.27, 1.17.28, 1.17.29, 1.17.30, 1.17.31, 1.17.32, 1.17.33, 1.17.34, 1.17.35, 1.17.36, 1.17.37, 1.17.38, 1.17.39, 1.17.40, 1.17.41, 1.17.42, 1.17.43, 1.17.44, 1.17.45, 1.17.46, 1.17.47, 1.17.48, 1.17.49, 1.17.50, 1.17.51, 1.17.52, 1.17.53, 1.17.54, 1.17.55, 1.17.56, 1.17.57, 1.17.58, 1.17.59, 1.17.60, 1.17.61, 1.17.62, 1.17.63, 1.17.64, 1.17.65, 1.17.66, 1.17.67, 1.17.68, 1.17.69, 1.17.70, 1.17.71, 1.17.72, 1.17.73, 1.17.74, 1.17.75, 1.17.76, 1.17.77, 1.17.78, 1.17.79, 1.17.80, 1.17.81, 1.17.82, 1.17.83, 1.17.84, 1.17.85, 1.17.86, 1.17.87, 1.17.88, 1.17.89, 1.17.90, 1.17.91, 1.17.92, 1.17.93, 1.17.94, 1.17.95, 1.17.96, 1.17.97, 1.17.98, 1.17.99, 1.17.100, 1.17.101, 1.17.102, 1.17.103, 1.17.104, 1.17.105, 1.17.106, 1.17.107, 1.17.108, 1.17.109, 1.17.110, 1.17.111, 1.17.112)
ERROR: No matching distribution found for boto3>=1.26.30 (from elastalert2)`

**Comment by nsano-rururu** [comment, marked as answer]
look
https://pypi.org/project/boto3/#history
